src/utils/utils_part.h

```diff
@@ -73,7 +74,27 @@ static inline void ocf_part_sort(struct ocf_cache *cache)
 	ocf_lst_sort(&cache->lst_part);
 }
 
-static inline ocf_cache_mode_t ocf_part_get_cache_mode(struct ocf_cache *cache,
+static inline bool ocf_part_is_enabled(struct ocf_request *req)
+{
+	return req->cache->user_parts[req->part_id].config->max_size != 0;
+}
+
+#define OCF_PART_SUCCESS	0
```

Comment:
How about renaming these constants to OCF_ALLOT_(SUCCESS/....) ?

Reply:
In current implementation no allocation takes place. How about OCF_PART_HAS_SPACE and OCF_PART_IS_FULL?

Reply:
looks good, thanks :)